lib/util.js

```diff
@@ -34,7 +34,7 @@ exports.sortFormats = (a, b) => {
 
   const getBitrate = (c) => {
     if (c.bitrate) {
-      let s = c.bitrate.split('-');
+      let s = c.bitrate.toString().split('-');
```

Comment:
is this a number now? if so, it wouldn't need to be `.split()`d anymore

Reply:
Sometimes, it's a number and sometimes not that's why i added `.toString()`